raft/consensus_module.nim

```diff
@@ -5,4 +5,95 @@
 #  * MIT license ([LICENSE-MIT](LICENSE-MIT))
 # at your option.
 # This file may not be copied, modified, or distributed except according to
-# those terms.
\ No newline at end of file
+# those terms.
+
+import types
+import protocol
+import log_ops
+import chronicles
+
+proc RaftNodeQuorumMin[SmCommandType, SmStateType](node: RaftNode[SmCommandType, SmStateType]): bool =
+  result = false
+  withRLock(node.raftStateMutex):
+    var cnt = 0
+    for peer in node.peers:
+      if peer.hasVoted:
+        cnt.inc
+    if cnt >= (node.peers.len div 2 + node.peers.len mod 2):
+      result = true
+
+proc RaftNodeHandleHeartBeat*[SmCommandType, SmStateType](node: RaftNode[SmCommandType, SmStateType], msg: RaftMessageAppendEntries): RaftMessageAppendEntriesResponse[SmStateType] =
```

Comment:
According to [NEP-1](https://nim-lang.org/docs/nep1.html), function names should use `camelCase` in Nim.

Reply:
Ok will fix it.